datadreamer/utils/luxonis_dataset_converter.py

```diff
@@ -32,7 +35,19 @@ def convert(self, dataset_dir, output_dir, split_ratios, copy_files=True):
         data = BaseConverter.read_annotations(annotation_path)
         self.process_data(data, dataset_dir, output_dir, split_ratios)
 
-    def process_data(self, data, dataset_dir, output_dir, split_ratios):
+    def process_data(self, data, dataset_dir, output_dir, split_ratios) -> None:
+        """Processes the data into LuxonisDataset format.
+
+        Args:
+        - data (dict): The data to process.
```

Comment:
Let's try to be consistent with docstring format. I believe in most files we use indents instead of `-`